tasks/main.yml

```diff
@@ -3,6 +3,9 @@
   apt: pkg=ntp={{ ntp_version }} state=present
 
 - name: Configure NTP
-  template: src=ntp.conf.j2 dest=/etc/ntp.conf
+  template: src={{ item }} dest=/etc/ntp.conf
+  with_first_found:
+    - "ntp.{{ ansible_distribution_release }}.conf.j2"
+    - "ntp.conf.default.j2"
```

Comment:
I'm debating whether it is better to just fail here vs. apply a default configuration file that we aren't confident will work. If the distribution release template doesn't exist, Ansible should fail hard.